app/src/main/java/com/lagradost/cloudstream3/utils/UIHelper.kt

```diff
@@ -123,6 +129,20 @@ object UIHelper {
         )
     }
 
+    fun clipboardHelper(label: UiText, text: CharSequence) {
+        try {
+            val clip = ClipData.newPlainText(label.asString(context!!), text)
```

Comment:
dont. do smth like 

context?.let { ctx -> 

}
or let ctx = context ?: return

Reply:
We never want to do !!